tests/integration/jpa/appl/pom.xml

```diff
@@ -0,0 +1,396 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.helidon.applications</groupId>
+        <artifactId>helidon-mp</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+        <relativePath>../../../../applications/mp/pom.xml</relativePath>
+    </parent>
+
+    <groupId>io.helidon.tests.integration.jpa</groupId>
+    <artifactId>helidon-tests-integration-jpa-appl</artifactId>
+    <name>Integration Tests: JPA MP Application</name>
+
+    <properties>
+        <mainClass>io.helidon.microprofile.cdi.Main</mainClass>
+        <mainClassH2>org.h2.tools.Server</mainClassH2>
+        <h2.db.version>1.4.197</h2.db.version>
+        <h2.db.password>p4ssw0rd</h2.db.password>
+        <h2.db.port>9092</h2.db.port>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <!-- we need this explicit version to work with native image -->
+                <groupId>com.h2database</groupId>
+                <artifactId>h2</artifactId>
+                <version>${h2.db.version}</version>
+            </dependency>
+            <dependency>
+                <!-- we need this explicit version to work with native image -->
+                <groupId>io.quarkus</groupId>
+                <artifactId>quarkus-jdbc-h2</artifactId>
+                <version>1.2.1.Final</version>
```

Comment:
Make sure to manage all versions with properties at the top.

Reply:
Those versions are native-image specific. maybe we can create some common definition for them to avoid copy-pasting.

Reply:
It does not matter if they are native-image specific.
- If this is actually something end-users should use then it should go under `dependencies/pom.xml`
- if this is specific to these JPA tests, then move it under the parent pom at `tests/integration/jpa/pom.xml`

Reply:
But this pom does not have `tests/integration/jpa/pom.xml` as parent. `It uses applications/mp/pom.xml`. I'll move versions there.

Reply:
Done.